spec/linter-eslint-spec.js

```diff
@@ -367,24 +380,14 @@ describe('The eslint provider for Linter', () => {
     let editor
     let didError
     let gotLintingErrors
-    let tempFixtureDir
-    let tempFixturePath
+    const tempFixtureDir = fs.mkdtempSync(tmpdir() + path.sep)
```

Comment:
Since this is the exact same in both test cases, you _could_ move it into `copyFileToTempDir`, if you want to leave it as a parameter though you might as well just call the function `copyFileToDir` 😛.